### ISSUE TITLE
Pirate Weather API unreachable since 5/26

### ISSUE THREAD

**Issue by teladog01** [alexander0042/pirateweather#73]
Hi there...I noticed several days ago that my app's weather data was not up to date. After looking through logs, I see that all calls to api.pirateweather.net have been failing since May 26. By failing, I mean I get no response at all and the call times out.

At first I chalked it up to a transient internet issue, but my ISP (Mediacom) couldn't see an issue on their end and said it must be a problem with the website. Then I noticed that the last entry on the [API Changelog](http://pirateweather.net/en/latest/changelog/) is dated May 26...same day calls started failing. Could be coincidence, but smells a little fishy :) 

Also, looking through the Issues I saw #64. I wonder if my problem accessing the site could be the same or similar issue?

EDIT: I can access pirateweather.net and merrysky.net without issue. This is strictly an issue with api.pirateweather.net.

**Comment by teladog01** [comment]
Apologies, my problem above turns out to NOT be a Pirate Weather issue. Rather, after many days troubleshooting I finally tracked it down to a recent OPNsense update that inexplicably started sending traffic to Pirate Weather API out over a VPN gateway by default. The other end of the VPN was just dropping these packets, so no responses were ever received. I've not yet tracked down what is causing this bizarre behavior in OPNsense, but at least I've found the problem and implemented a workaround.

Very sorry to bother...
